package.json

```diff
@@ -34,7 +35,9 @@
     "dep-graph": "nx dep-graph",
     "help": "nx help",
     "storybook:island-ui": "nx run island-ui-storybook:storybook",
-    "ts-node": "node -r ./scripts/ts-node"
+    "ts-node": "node -r ./scripts/ts-node",
+    "schemas": "test -f libs/api/schema/src/lib/schema.d.ts || echo 'export default () => {}' >> libs/api/schema/src/lib/schema.d.ts && yarn nx run api:contentful-types && yarn affected:schemas",
```

Comment:
That's a bit ugly but the thing is `libs/api/schema/src/lib/schema.d.ts` is ignored, even though, without it, we can't build the API, because the file is required for some [import](https://github.com/island-is/island.is/blob/master/libs/api/schema/src/index.ts#L1). So if the file doesn't exist I created it with an empty function, and it gets overwritten once the codegen is done.

I will tackle that issue in another PR to remove this file and move all schema locally to each apps, so we won't depend on shared schemas for the apps. It already has been done for the [web project](https://github.com/island-is/island.is/blob/master/apps/web/codegen.yml)